apps/modernization-ui/src/apps/landing/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@trussworks/react-uswds';
+import { AlertBanner } from 'apps/page-builder/components/AlertBanner/AlertBanner';
+import classNames from 'classnames';
+import styles from './signIn.module.scss';
+import { NavLinkButton } from 'components/button/nav/NavLinkButton';
+import { WelcomProps } from 'welcomeLayout';
+
+export const SignIn = ({ handleWelcomeEvent }: WelcomProps) => {
+    return (
+        <div className="">
+            <h2 className={classNames(styles.heading)}>Login</h2>
+            <div className={classNames(styles['alert-banner'])}>
+                <AlertBanner type="info">
+                    <p>
+                        Please be sure to avoid entering any real PHI/PIl data on the demo site. All information entered
+                        will be viewable by other users.
+                    </p>
+                </AlertBanner>
+            </div>
+            <NavLinkButton to="/login" className="margin-top-2">
```

Comment:
issue:  The route `/login` is for development use only.  This should be pointing to the NBS 6 Login page at `/nbs/login`